src/errors.rs

```diff
@@ -8,6 +8,9 @@ use std::sync::Arc;
 /// Error types within Rahmen
 #[derive(std::fmt::Debug)]
 pub enum RahmenError {
+    /// unknown case for conversion
+    /// TODO it's NYI
```

Comment:
I see the variant being used elsewhere. What part is NYI?

Reply:
IDK, must have been left over.